### DISCUSSION TITLE
Simple calendar view

### DISCUSSION THREAD

**Discussion by Akselmo** [InfiniTimeOrg/InfiniTime#1529] [Ideas]
### Verification

- [X] I searched for similar feature request and found none was relevant.

### Pitch us your idea!

Calendar view that shows the current month and current day is highlighted

### Description

I just would like to see quickly from my watch what day of the month it is, or when discussing future plans it would be quicker than looking from phone. Eventually this could perhaps be used as a base for adding support for events, but right now I'd just like to see the current month and the current day highlighted. 

Also being able to scroll left and right to switch months would be nice.

**Comment by Riksu9000** [comment]
I'm sure a calendar with events will come at some point. However most of the watchfaces already show the current month and day, so do you think a view of an empty calendar might still be useful in this case?

**Comment by Akselmo** [comment]
I think it would, just for quickly checking what some day of some month is. "Okay next months 20th is monday, etc etc." Although if calendar with events is coming someday, making this now may be unnecessary..

**Comment by medeyko** [comment]
Definitely yes, a simple calendar view for current and next month, maybe with inclusion of some additional info like number of a week, full moon days, solstice, etc. is useful for a lot of people (me included, hehe).

I think it would be wise to start with this simple task, and later maybe integrate events.

**Comment by thnikk** [comment]
I've implemented a simple calendar view with PR #923

**Comment by rimugu** [comment]
Any chance we see this in a release?
